code/demo/android-native/app/build.gradle

```diff
@@ -1,37 +1,59 @@
 apply plugin: 'com.android.application'
 
 android {
-    compileSdkVersion 28
+  buildToolsVersion = '30.0.2'
```

Comment:
I think that line can be dropped, and the latest build tools will be used

Reply:
You're right